lib/drm/drm_engine.js

```diff
@@ -2668,6 +2637,33 @@ shaka.drm.DrmEngine = class {
     this.newInitData('cenc', combinedData);
     return this.allSessionsLoaded_;
   }
+
+  /**
+   * Create a DrmInfo using configured clear keys and assign it to each variant.
+   * Only modify variants if clear keys have been set.
+   * @see https://bit.ly/2K8gOnv for the spec on the clearkey license format.
+   *
+   * @param {!Object.<string, string>} configClearKeys
+   * @param {!Array.<shaka.extern.Variant>} variants
+   */
+  static configureClearKey(configClearKeys, variants) {
+    const clearKeys = shaka.util.MapUtils.asMap(configClearKeys);
+    if (clearKeys.size == 0) {
+      return;
+    }
+    const clearKeyDrmInfo =
+        shaka.util.ManifestParserUtils.createDrmInfoFromClearKeys(clearKeys);
+    if (clearKeyDrmInfo) {
```

Comment:
It doesn't look like there's any situation where `shaka.util.ManifestParserUtils.createDrmInfoFromClearKeys` returns null, so you probably don't need this "if" condition anymore.

Actually, looking at that utility method, its header should probably say `@return {!shaka.extern.DrmInfo}` instead of `@return {shaka.extern.DrmInfo}`, to make it clearer the return value is non-null...

Reply:
done!